src/lib/modules/Document/DocumentPublicationInfo.js

```diff
@@ -19,43 +20,43 @@ export class DocumentPublicationInfo extends Component {
                 {idx > 0 && <Divider horizontal />}
                 <Table definition key={publication.journal_title}>
                   <Table.Body>
-                    {!_isEmpty(publication.journal_title) && (
+                    {publication?.journal_title && (
```

Comment:
Can the `_isEmpty` function be removed from the import or is it used somewhere else in this file?

Reply:
it is :smile: